bumble/host.py

```diff
@@ -92,7 +92,19 @@ def __init__(self, controller_source = None, controller_sink = None):
         if controller_sink:
             self.set_packet_sink(controller_sink)
 
+    async def flush(self):
```

Comment:
is there a need for `flush` other than in a reset? If not, maybe the event should be called `reset`, and move that logic inside the `reset()` method.

Reply:
Yes we need `flush` outside of the `reset`